src/phoenix/pointcloud/projectors.py

```diff
@@ -25,6 +19,11 @@ class Umap:
     min_dist: float = 0.1
 
     def project(self, mat: Matrix, n_components: int) -> Matrix:
+        with warnings.catch_warnings():
+            from numba.core.errors import NumbaWarning
+
+            warnings.simplefilter("ignore", category=NumbaWarning)
+            from umap import UMAP
```

Comment:
delayed import could save 3s when they're not needed

<img width="233" alt="Screenshot 2024-09-29 at 6 01 00 PM" src="https://github.com/user-attachments/assets/fc0024d2-36ab-428f-b3cf-a0ed4e8e8ade">